src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -1032,4 +1036,46 @@ public function handleEmote(EmotePacket $packet) : bool{
 		$this->player->emote($packet->getEmoteId());
 		return true;
 	}
+
+	public function handleRequestAbility(RequestAbilityPacket $packet) : bool{
```

Comment:
Why is this function present?

Reply:
It was removed after the pull request creation, it will go away when I rebase.
https://github.com/pmmp/PocketMine-MP/commit/fe3e2cc90a002b6a19e4b515d01f12dd9855cfc9#diff-f405c0ec75b0721cb66a250d6938b9267fd68285ffa443dfe8e0e156afac04e0L1035

Reply:
Update, I have no idea what's happening and it did not go away when I rebased